coremltools/converters/mil/frontend/torch/utils.py

```diff
@@ -0,0 +1,39 @@
+#  Copyright (c) 2024, Apple Inc. All rights reserved.
+#
+#  Use of this source code is governed by a BSD-3-clause license that can be
+#  found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+from enum import Enum
+
+_DEFAULT_OP_NAMESPACES = set(["aten", "prim"])
+
+
+class TorchFrontend(Enum):
+    TORCHSCRIPT = 1
+    EXIR = 2
+
+
+def sanitize_op_kind(op_kind: str) -> str:
+    # We conventionally skip the aten/prim namespaces in our naming
```

Comment:
Why do we skip aten/prim?

Also I'd like to see a doctoring for this function.